crates/core/app/src/server/info.rs

```diff
@@ -259,6 +259,7 @@ impl Info {
                         channel_id: chan_id,
                         port_id: PortId::transfer(),
                         channel_end: channel,
+                        upgrade_sequence: 0,
```

Comment:
New field; I did not know what to do with it, so opted for the default value (as per protobuf convention).